src/control/test/pp_test.cpp

```diff
@@ -113,4 +116,161 @@ TEST(PurePursuitTests, Test_pp_steering_control_law_1) {
   //  Alpha: 0.804189
   //  ld_: 5.38516
   EXPECT_NEAR(0.381, steering_cmd, 0.001);
-}
\ No newline at end of file
+}
+
+/**
+ * @brief Test PurePursuit - update_closest_point()
```

Comment:
Maybe give some more insight into the scenario being tested, even if it is just one